src/main/scala/uk/gov/hmrc/ct/box/ValidatableBox.scala

```diff
@@ -238,6 +240,13 @@ trait ValidatableBox[T <: BoxRetriever] {
     }
   }
 
+  def validateStringMaxLength(boxId: String, value: String, max: Int): Set[CtValidation] = {
```

Comment:
You have an overloaded version of this which takes an option and then you can check for None, instead of doing getOrElse("")